README.md

```diff
@@ -24,13 +40,16 @@ Install with Homebrew:
 brew install stellar/tap/stellar-cli
 ```
 
-## Install Experimental Features
+## Installation with Experimental Features
 To use the potentially unreleased bleeding edge CLI functionalities, install from git:
 ```
-cargo install --locked stellar-cli --features opt --git git@github.com:stellar/stellar-cli.git
+cargo install --locked stellar-cli --features opt --git https://github.com/stellar/stellar-cli.git
 ```
 
-## Setup Autocomplete
+## Autocomplete
+The Stellar CLI supports some basic autocompletion. To set up autocomplete, run
```

Comment:
```suggestion
The Stellar CLI supports some autocompletion. To set up, run
```
It's pretty good just not the auto-generated one for an invoke, but otherwise works well!